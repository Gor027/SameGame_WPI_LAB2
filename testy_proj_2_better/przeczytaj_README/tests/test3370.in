E
EE"EE ""    "  
E  E EE"E  E  EE
"EEEEEEEEE"" " "E E"  E""""E""  ""      E""" E   EE"E"EE E
E   E"  E  E EE"""""E   " ""     " "E "E EE"E"    E "    E""E E""EEE" "E """EE""
"""EEE E"""""E"EE  "EEE"E "" 
E"   
E"E"E" "E"""E" E"    "
 EEEE    " 
EE "EEE"E"EE E E EEE"E""E E "  " "EE E   
