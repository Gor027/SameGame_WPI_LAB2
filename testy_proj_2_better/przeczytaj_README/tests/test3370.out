E         E   E
EE        " " "
EE    " ""E E "E  "    " "
""    """"""EEE" E"EEE"" "E    "
E"""" "E"""""EEE"""E"E"E """   " "     """ E E E   E
"""""""E"EEEE"E"EEE"E""E"E""""""EEEE"EEEE"E"EE"""EEE""EE""EEE""E"""EE""
